code_generator_funcadl_xAOD/pyproject.toml

```diff
@@ -8,7 +8,7 @@ packages = [{ include = "code_generator_funcadl_xaod" }]
 
 [tool.poetry.dependencies]
 python = "~3.10"
-func-adl-xAOD = "2.2.0"
+func-adl-xAOD = "^2.2.0"
```

Comment:
I'm a bit confused here - doesn't this syntax allow 2.2.0 to be used? Don't you actually want ^2.2.1 ? (or >=2.2.1)